postgresql/restore.sh

```diff
@@ -6,13 +6,13 @@
 
 # Before running the restore, you'll have to stop the ECS task to close any open connections.
 # Then, run the following to drop the database.
-# psql -U concordia -h $POSTGRES_HOST
+# psql -U concordia -h $POSTGRESQL_HOST
 # \c postgres
 # drop database concordia;
 
-export POSTGRES_HOST=
+export POSTGRESQL_HOST=localhost
 export DUMP_FILE=concordia.dmp
 
-pg_restore --create --clean -U concordia -h $POSTGRES_HOST -Fc --dbname=postgres --no-owner --no-acl $DUMP_FILE
+pg_restore --create --clean -U concordia -h $POSTGRESQL_HOST -Fc --dbname=postgres --no-owner --no-acl $DUMP_FILE
```

Comment:
Since we're in here, I'd also change this to get a clean [`shellcheck`](https://www.shellcheck.net/) run:

```suggestion
pg_restore --create --clean -U concordia -h "${POSTGRESQL_HOST}" -Fc --dbname=postgres --no-owner --no-acl $DUMP_FILE
```